Content.Server/NodeContainer/Nodes/PipeNode.cs

```diff
@@ -24,6 +24,12 @@ public partial class PipeNode : Node, IGasMixtureHolder, IRotatableNode
         [DataField("pipeDirection")]
         public PipeDirection OriginalPipeDirection;
 
+        /// <summary>
+        ///     The *current* layer to which the pipe node is assigned.
+        /// </summary>
+        [DataField("pipeLayer")]
+        public byte CurrentPipeLayer;
```

Comment:
I wonder if this should remain a `byte`, or if it should gain a new `PipeLayer` type (like the above `PipeDirection`).